src/tools/FreehandRoiSculptorTool.js

```diff
@@ -252,12 +246,14 @@ export default class FreehandRoiSculptorTool extends BaseTool {
       handleRadius: radiusCanvas,
     };
 
-    drawHandles(
-      context,
-      eventData,
-      this.configuration.mouseLocation.handles,
-      options
-    );
+    if (this.configuration.drawHandles) {
+      drawHandles(
```

Comment:
I wasn't aware that the sculptor hand handles? It's not a persistent annotation, so we likely don't need this?

Reply:
👍  I have removed the conditional along with those other Roi tools mentioned here.